modules/core/shared/src/main/scala/lucuma/core/enums/CloudExtinction.scala

```diff
@@ -6,29 +6,30 @@ package lucuma.core.enums
 import lucuma.core.util.Display
 import lucuma.core.util.Enumerated
 
-sealed abstract class CloudExtinction(val toDeciBrightness: Int) extends Product with Serializable {
+sealed abstract class CloudExtinction(val tag: String, val toDeciBrightness: Int) extends Product with Serializable {
   def toBrightness: Double = toDeciBrightness / 10.0
   def label: String        = f"< $toBrightness%.1f mag"
 }
 
 object CloudExtinction {
-  case object PointOne       extends CloudExtinction(1)
-  case object PointThree     extends CloudExtinction(3)
-  case object PointFive      extends CloudExtinction(5)
-  case object OnePointZero   extends CloudExtinction(10)
-  case object OnePointFive   extends CloudExtinction(15)
-  case object TwoPointZero   extends CloudExtinction(20)
-  case object ThreePointZero extends CloudExtinction(30)
+  case object PointOne       extends CloudExtinction("point_one", 1)
+  case object PointThree     extends CloudExtinction("point_three", 3)
+  case object PointFive      extends CloudExtinction("point_five", 5)
```

Comment:
i'm tempted to make these `ZeroPointOne` etc to be consistent with the `OnePoint...` `TwoPoint...`, and `ThreePoint...`s. But we'd need to change the schema and patch up tmp-api accordingly.

Reply:
Maybe come back to this once we're off tmp-api?

Reply:
Sounds good.